src/app/(auth)/register/page.tsx

```diff
@@ -0,0 +1,40 @@
+import { register } from "@/auth/register";
+
+export default function RegisterPage() {
+  return (
+    <div className="absolute left-[50%] top-[50%] w-[25vw] -translate-x-2/4 -translate-y-1/2 justify-center rounded-md bg-white/25 p-10">
```

Comment:
the same applies here, also you can just put layout.tsx file inside (auth) directory and put one wrapper there for login page and signup page to share